### DISCUSSION TITLE
[View Trades Config] Add maximum resources setting

### DISCUSSION THREAD

**Discussion by trs4ece** [BlackAsLight/DocScripts#6] [Ideas]
This setting is more for those who are trying to maintain a maximum amount of resources. Although we already have a setting for the maximum amount of resources for sale on the market, there is not a setting that lets you retain a maximum number of a resource.

Use case:
I would like to retain at most 10 days of oil. This works out to 15,750 units. I currently have 992 units and therefore need 14,758 more units.
1. When I try to purchase oil, I would like the number of units to auto-populate with 14,758 rather than the maximum number of units that I could purchase with my current bankroll.
2. If I already have 758 units of buy orders out, then when I try to directly purchase more or create a new buy-order, I would like that value automatically set to 14,000.

**Comment by BlackAsLight** [comment]
I've currently and slowly been rewriting this script from js to tsx and actually came up with this idea a few days earlier. For the most part it's already been implemented. It will be called Max "Resource" on the sidebar, and the current Max "Resource" is being renamed to Max "Resource"/Offer I think.
